libs/ts-bundler/src/loader/bootstrap.ts

```diff
@@ -0,0 +1,15 @@
+import * as E from "./embassy.ts";
+import * as T from "./types.d.ts";
+type Maybe<T> = T | undefined;
```

Comment:
```suggestion
export * from './embassy.ts'
type Maybe<T> = T | undefined;
```

Reply:
Do the later exports override the ones with the same names from ./embassy.ts?

Reply:
Yes, the later exports override.